src/nvme/fabrics.c

```diff
@@ -417,6 +432,8 @@ static int build_options(nvme_host_t h, nvme_ctrl_t c, char **argstr)
 			 nvme_ctrl_get_trsvcid(c)) ||
 	    (hostnqn && add_argument(argstr, "hostnqn", hostnqn)) ||
 	    (hostid && add_argument(argstr, "hostid", hostid)) ||
+	    (discover && !discovery_nqn &&
```

Comment:
This is good. It allows operating with older kernels that don't support the new "discovery" attribute as long as we use the well-known NQN (`nqn.2014-08.org.nvmexpress.discover`). I was wondering how you would address that. :grinning: